templates/gatsby-drupal/.platform.template.yaml

```diff
@@ -0,0 +1,65 @@
+
+  # An image URI (either base64-encoded or a URL) representing the template.
+
+  # Additional notes displayed in the template's detail view.
+  # Each note object is displayed as a small section heading with content below. Supports limited HTML.
+  notes:
+    - heading: "Features"
+      content: |
+          Node.js 12<br />
+          PHP 7.4<br/>
+          MariaDB 10.4<br/>
+          Redis 5.0<br/>
+          Automatic TLS certificates<br />
+          npm-based build for Gatsby<br />
+          Composer-based build for Drupal<br />
+          Multi-app configuration<br />
+          Delayed SSG build (post deploy hook)<br />
+
+
+# This key describes the initialization call made to the master environment at
+# project creation time. This is part of the full v2 UI operation mode, which
+# places project schema/options selection early in the creation process, rather
+# than later as it exitss now. To allow this schema to be backwards-compatible,
+# this key also gets mapped to the appropriate location in project.settings so
+# that the current UI can have its own workflow overridden as well.
+initialize:
+  repository: git://github.com/platformsh-templates/gatsby-drupal.git@master
+  config: null
+  files: []
+  profile: Gatsby with Drupal
```

Comment:
Let's go ahead and add the code here to forbid Small plans, since we know it won't work anyway.